workflow/usage.rst

```diff
@@ -237,6 +237,12 @@ This means that each event has access to the following information:
 :method:`Symfony\\Component\\Workflow\\Event\\Event::getTransition`
     Returns the :class:`Symfony\\Component\\Workflow\\Transition` that dispatches the event.
 
+:method:`Symfony\\Component\\Workflow\\Event\\Event::getWorkflowName`
+    Returns a string with the name of the workflow that triggered the event.
+
+.. versionadded:: 3.3
+    The ``getWorkflowName()`` method was introduced in Symfony 3.3.
```

Comment:
I would indent the `versionadded` directive so that it is treat as part of the definition.